### DISCUSSION TITLE
Import of raster maps

### DISCUSSION THREAD

**Discussion by claus2022** [osmandapp/OsmAnd#15446] [Maps Related]
I have a large collection of calibrated raster maps, which I generated with the great Fugawi software (which is unfortunately not longer sold). The map-format is fx4 (Fugawi) or pth (Pathaway) and they can de converted with various tools into GeoTif, GeoPdf or other calibrated raster map formats.  Is there any way, to import such raster maps into OsmAnd?

**Comment by barteis** [comment]
If you have access to a Windows machine, MAPC2MAPC might be the way to go on converting your map files. More on it and OsmAnd here: https://youtu.be/Y_fekLfcUOc.  
OsmAnd supports neither GeoPDF nor GEOtiff.

**Reply by claus2022** [reply]
Thank you very much. So far, this worked fine. I converted my raster maps into sqlite-files. One question is remaining: What are the steps to import these sqlite files into the OsmAnd app?

**Reply by barteis** [reply]
I thought I knew, but no longer do.  Transferring the maps can be done in the usual ways, including connecting a cable between your PC and your device.  But due to changes within Android, I'm not sure which directory they go in--or whether you have to create one.  Maybe someone else reading this can help.

**Comment by mariush444** [comment]
till android 11.0 you can move/create osmand directory in Android/media instead Android/data then you have access to the folders as previously

**Reply by claus2022** [reply]
I have the pixel 5 with Android 13. No chance to import custom maps?

**Reply by mariush444** [reply]
I didn't use Android 13 yet, so I don't know. If you have access to Android/media folder then you can do it in my opinion. 
If you have no access to this folder then you can't.

I guess that other possibility is to make something similar to plug in "map legend" or "travel guide". The prepared file should be unpacked to needed folders. But I didn't test this method yet.
